modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/XpanseTestApplication.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.orchestrator;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.scheduling.annotation.EnableAsync;
+
+/**
+ * Main entry class to xpanse runtime. This class can be directly executed to start the server.
+ */
+@SpringBootApplication
+@EnableAsync(proxyTargetClass = true)
+@EnableJpaRepositories(basePackages = "org.eclipse.xpanse")
+@EntityScan(basePackages = "org.eclipse.xpanse")
+@ComponentScan(basePackages = "org.eclipse.xpanse")
+@EnableJpaAuditing
+public class XpanseTestApplication {
```

Comment:
is it not possible to simply start the actual main class here with test profile?

Reply:
The actual main class cannot be started here, possible cyclic dependency.

Reply:
oh ok. yes you are right. then can we move this test directly under runtime package?
If not, i would avoid using this and instead only use other ways to instantiate classes which are needed.

Reply:
yes ok,move to the runtime package is good, i will do this.